Spanner/tests/Unit/SpannerClientTest.php

```diff
@@ -200,6 +200,93 @@ public function testCreateInstance()
         $this->assertInstanceOf(LongRunningOperation::class, $operation);
     }
 
+    /**
+     * @group spanner-admin
+     */
+    public function testCreateInstanceWithNodes()
+    {
+        $this->connection->createInstance(Argument::that(function ($arg) {
+            if ($arg['name'] !== InstanceAdminClient::instanceName(self::PROJECT, self::INSTANCE)) {
+                return false;
+            }
+
+            if ($arg['config'] !== InstanceAdminClient::instanceConfigName(self::PROJECT, self::CONFIG)) {
+                return false;
+            }
+
+            return isset($arg['nodeCount']) && $arg['nodeCount'] === 2;
+        }))
+            ->shouldBeCalled()
+            ->willReturn([
+                'name' => 'operations/foo'
+            ]);
+
+        $this->client->___setProperty('connection', $this->connection->reveal());
+
+        $config = $this->prophesize(InstanceConfiguration::class);
+        $config->name()->willReturn(InstanceAdminClient::instanceConfigName(self::PROJECT, self::CONFIG));
+
+        $operation = $this->client->createInstance($config->reveal(), self::INSTANCE, [
+            'nodeCount' => 2
+        ]);
+
+        $this->assertInstanceOf(LongRunningOperation::class, $operation);
+    }
+
+    /**
+     * @group spanner-admin
+     */
+    public function testCreateInstanceWithProcessingUnits()
+    {
+        $this->connection->createInstance(Argument::that(function ($arg) {
+            if ($arg['name'] !== InstanceAdminClient::instanceName(self::PROJECT, self::INSTANCE)) {
+                return false;
+            }
+
+            if ($arg['config'] !== InstanceAdminClient::instanceConfigName(self::PROJECT, self::CONFIG)) {
+                return false;
+            }
+
+            return isset($arg['processingUnits']) && $arg['processingUnits'] === 2000;
+        }))
+            ->shouldBeCalled()
+            ->willReturn([
+                'name' => 'operations/foo'
+            ]);
+
+        $this->client->___setProperty('connection', $this->connection->reveal());
+
+        $config = $this->prophesize(InstanceConfiguration::class);
+        $config->name()->willReturn(InstanceAdminClient::instanceConfigName(self::PROJECT, self::CONFIG));
+
+        $operation = $this->client->createInstance($config->reveal(), self::INSTANCE, [
+            'processingUnits' => 2000
+        ]);
+
+        $this->assertInstanceOf(LongRunningOperation::class, $operation);
+    }
+
+    /**
+     * @group spanner-admin
+     */
+    public function testCreateInstanceRaisesInvalidArgument()
+    {
+        if (method_exists($this, 'setExpectedException')) {
```

Comment:
There is an annotation which can be applied and should do the trick for PHP 5.5+: https://github.com/googleapis/google-cloud-php/blob/8611b2337883fb3693d5295d5364059f9f223c77/Bigtable/tests/Unit/Filter/Builder/RangeTest.php#L45